scalacheck-binding/src/main/scala/scalaz/scalacheck/ScalazArbitrary.scala

```diff
@@ -533,6 +533,9 @@ object ScalazArbitrary extends ScalazArbitraryPlatform {
   implicit def lazyEitherTArb[F[_], A, B](implicit A: Arbitrary[F[LazyEither[A, B]]]): Arbitrary[LazyEitherT[F, A, B]] =
     Functor[Arbitrary].map(A)(LazyEitherT[F, A, B](_))
 
+  implicit def constIdTArbitrary[A, F[_]](implicit A: Arbitrary[A], APP: Applicative[F]): Arbitrary[IdT[F, A]] =
+    Functor[Arbitrary].map(A)(a => IdT[F, A](APP.pure(a)))
```

Comment:
Why not `implicit def idTArbitrary[A, F[_]](implicit A: Arbitrary[F[A]]): Arbitrary[IdT[F, A]]` ?